docs/usage.rst

```diff
@@ -200,3 +200,27 @@ This is the case when you choose to integrate your user registration process wit
 Otherwise, serializer classes for ``verify`` and ``register`` views will not be available.
 
 3. Latest ``security_code`` generated for a ``phone_number`` can be found at ``/admin/phone_verify/smsverification/`` URL.
+
+4. ``phone_verify.models.SMSVerification`` emits ``post_save`` signal on its creation and updation. Suppose you want to send an email, informing the user that his/her phone number has been successfully verified. We can implement that by creating a receiver which would listen to ``post_save`` signal of ``SMSVerification`` model. For example:
+
+.. code-block:: python
+
+    # In signals.py
+    from phone_verify.models import SMSVerification
+    from django.db.models.signals import post_save
+    from django.dispatch import receiver
+    from django.core.mail import send_mail
+
+
+    # `send_phone_verify_email` function will get fired when a new entry is created or the model instance is updated
+    @receiver(post_save, sender=SMSVerification)
+    def send_phone_verify_email(sender, instance, **kwargs):
+        # Check if the instance is verified or not.
+        if instance.is_verified:
+            send_mail(
+                'Subject here',
```

Comment:
Also, a tip: if you're using a function call, it makes sense to use keyword arguments so that people reading your code can immediately figure out which parameter expects what.